### ISSUE TITLE
Square bracket not printed correctly when using a german keyboard layout

### ISSUE THREAD

**Issue by seekM** [talonvoice/talon#230]
`"square": "["` from the [starter pack](https://github.com/lunixbochs/talon_starter_pack/blob/master/misc/basic.py#L22) prints a `[` when an english keyboard layout is used, but `8` with a german keyboard layout. Other characters aren't outputted correctly as well, e.g. `backslash` yields `ß` with a german keyboard layout activated. Tested on windows with DNS 13 and an english language profile.

**Comment by AndreasArvidsson** [comment]
Same here using a Swedish layout, but not for all applications. Most applications like: notepad, VScode, Firefox all works fine, but several CLI as git bash, WSL and others wont display some brackets and show other symbols or press other keys. Like brace deletes in WSL. Changing to an English layout before starting Talon fixes the issue.

**Comment by lunixbochs** [comment]
Are all of the failing keys using AltGr? Does it happen anywhere besides Console Host apps (Windows Terminal, cmd, git bash)?

**Comment by AndreasArvidsson** [comment]
Only in CLI as far as I know, but I think it figured out why to some extent. The problem with windows terminal is that ctrl-alt-NUMBER is bound to change tab. So when I (with talon) say brace talon sends ctrl-alt-7 and windows terminal switches to tab 7 instead of writing the actual brace. Once I change the binding in windows terminal to something else so that ctrl-alt-NUMBER is free I could use talon with symbols again.
I also did some testing with virtual keys and SendInput() in c and there is no altgr key to actually send to my knowledge.

**Comment by lunixbochs** [comment]
closing as duplicate of https://github.com/talonvoice/talon/issues/164, which seems to identify the underlying issue better

**Comment by lunixbochs** [comment]
This might be fixed now on the latest beta.
